app/portainer/services/api/endpointService.js

```diff
@@ -16,6 +16,10 @@ function EndpointServiceFactory($q, Endpoints, FileUploadService) {
     return Endpoints.snapshot({}, {}).$promise;
   };
 
+  service.checkEndpointStatus = function(endpointId) {
```

Comment:
Seems redundant with the usage of `SystemService.ping`. What's the point of wrapping this function? Can't you use `SystemService.ping()` instead of `EndpointService.checkEndpointStatus()` ?